File: core/src/main/java/com/pluxity/facility/station/dto/StationCreateRequest.java
package com.pluxity.facility.station.dto;

import com.pluxity.facility.facility.dto.FacilityCreateRequest;
import com.pluxity.facility.floor.dto.FloorRequest;
import java.util.List;

public record StationCreateRequest(
        FacilityCreateRequest facility, List<FloorRequest> floors, List<Long> lineIds) {}

File: core/src/main/kotlin/com/pluxity/station/dto/StationCreateRequest.kt
package com.pluxity.station.dto

import com.pluxity.facility.dto.FacilityCreateRequest
import com.pluxity.facility.floor.dto.FloorRequest

data class StationCreateRequest(
    val facility: FacilityCreateRequest,
    val floors: List<FloorRequest> = emptyList(),
    val lineIds: List<Long> = emptyList(),
    val stationCodes: List<String> = emptyList(),
)
